src/lib/analytics/datadog.ts

```diff
@@ -0,0 +1,23 @@
+import { datadogLogs } from '@datadog/browser-logs';
+
+const CLIENT_TOKEN = import.meta.env.VITE_DATADOG_CLIENT_TOKEN;
```

Comment:
thanks for flagging! right now this is how i'm envisioning it:
- one client token for dydx internal use. this will be shared across dev/testnet/mainnet and will be flagged by environment.
- every deployer who wants to opt into feeding us their dd tracking will have their own unique client token

AFAIK client tokens are only really used to enable RUM/analytics/logging unlike api keys where you might have a bunch of different rules/permissions/workspaces etc.